datadog_checks_base/datadog_checks/base/utils/db/transform.py

```diff
@@ -43,19 +45,32 @@ def get_tag(transformers, column_name, **modifiers):
     to the string `true` or `false`. So for example if you named the column `alive` and the result was the
     number `0` the tag will be `alive:false`.
     """
-    template = '{}:{{}}'.format(column_name)
     boolean = is_affirmative(modifiers.pop('boolean', None))
 
-    def tag(_, value, **kwargs):
-        # type: (List, str, Dict[str, Any]) -> str
+    def tag(_, value, check, **kwargs):
+        # type: (List, str, AgentCheck, Dict[str, Any]) -> List[str]
         if boolean:
             value = str(is_affirmative(value)).lower()
-
-        return template.format(value)
+        return _transform_tag(name, value, check)
 
     return tag
 
 
+def _transform_tag(name, value, check):
```

Comment:
I think we should not modify the database utils at all and instead add your logic here: https://github.com/DataDog/integrations-core/blob/d96a2de3ba31176060546789597429d727906f7f/datadog_checks_base/datadog_checks/base/checks/base.py#L1068-L1075